src/universal-app/BUILD.bazel

```diff
@@ -0,0 +1,57 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_test")
+load("@io_bazel_rules_sass//:defs.bzl", "sass_binary")
+load("//:packages.bzl", "CDK_EXPERIMENTAL_TARGETS", "CDK_TARGETS", "GOOGLE_MAPS_TARGETS", "MATERIAL_EXPERIMENTAL_SCSS_LIBS", "MATERIAL_EXPERIMENTAL_TARGETS", "MATERIAL_TARGETS")
+load("//tools:defaults.bzl", "ng_module", "ts_library")
+
+ng_module(
+    name = "kitchen-sink",
+    srcs = [
+        "kitchen-sink-mdc/kitchen-sink-mdc.ts",
+        "kitchen-sink-root.ts",
+        "kitchen-sink/kitchen-sink.ts",
+    ],
+    assets = [
+        "kitchen-sink/kitchen-sink.html",
+        "kitchen-sink-mdc/kitchen-sink-mdc.html",
+    ],
+    deps = [
+        "@npm//@angular/platform-server",
+    ] + CDK_TARGETS + CDK_EXPERIMENTAL_TARGETS + MATERIAL_TARGETS + GOOGLE_MAPS_TARGETS + MATERIAL_EXPERIMENTAL_TARGETS,
```

Comment:
I don't think you need the google maps targets

Reply:
Yep. For some reason I wanted to add it for consistency.. but since we don't need it, we can be more fine-grained :+1: